terracommon/datastore/serializers.py

```diff
@@ -29,7 +29,3 @@ class RelatedDocumentPDFSerializer(RelatedDocumentSerializer):
 
 class RelatedDocumentFileSerializer(RelatedDocumentSerializer):
```

Comment:
Isn't this serializer used anywhere ?

Reply:
Well it could be used one day... but yes, not at the moment